h/services/rename_user.py

```diff
@@ -1,5 +1,4 @@
 from h import models
-from h.search import index
```

Comment:
Could this be called `user_rename.py` instead? As we also have:

* `user.py`
* `user_signup.py`
* `user_unique.py`
* `user_update.py`

At least it's in good company with `delete_user.py`...

Also topic for another day, but when you repeat something that much, it might be time for a directory.